src/content/contributing/style-guide/index.md

```diff
@@ -0,0 +1,188 @@
+
+Because [Ethereum.org](http://ethereum.org) caters to a wide variety of users, all content on the site should make an effort to explain any technologies and concepts as simply as possible.
+
+## Objectivity {#objectivity}
+
+[Ethereum.org](http://ethereum.org) documentation aims to maintain a credibly neutral source of truth to inform readers about Ethereum and its ecosystem. Some examples of things that we don't want in the content on Ethereum.org:
+
+**Grand, unverifiable claims about Ethereum or adjacent technologies**
+
+> e.g. _ "Ethereum will take over the world because..."_
+
+**Hostile or confrontational language aimed at any organization or person**
+
+> e.g. _ "Company X is bad because they are centralized!"_
+
+**Politically charged rhetoric**
+
+> e.g. _ "This political party are better for decentralization because..."_
+
+## Tone {#tone}
+
+The tone of the content on the site should be welcoming, friendly and authoritative. Where possible, jargon should be minimized and simple language used instead.
+
+## Best Practices {#best-practices}
+
+### Use American English {#american-english}
+
+For words that have multiple spellings, use American English over British English.
+
+**For example:**
+
+- `decentralized` over `decentralised`
+- `color` over `colour`
+- `analyze` over `analyse`
+
+### Acroynyms {#acroynyms}
+
+When introducing an unfamiliar acronym, spell out the full term, and put the acronym in parentheses. Put both the full term and acronym in bold.
+
+**For example:**
+
+"Ethereum, like Bitcoin, currently uses a consensus protocol called **Proof-of-work (PoW)**."
+
+### Consistency {#consistency}
+
+Many of the topics covered on [Ethereum.org](http://ethereum.org) are technically complex. To reduce confusion to the reader, terms should be used consistently. For example, don't cycle back-and-forth between proof-of-work and PoW at random.
+
+### Mainnet {#mainnet}
+
+When referring to the Ethereum Mainnet (i.e. not referring to a testnet) use the proper noun. Proper nouns help avoid confusion and build greater understanding.
+
+**Correct usage:**
+
+- Mainnet
+- Ethereum Mainnet
+
+**Incorrect usage:**
+
+- main net
+- mainnet
+- Main net
+- Ethereum mainnet
+
+### Proof-of-work {#proof-of-work}
+
+Proof-of-work should be capitalized at the beginning of a sentence or when the term is introduced for the first time on a page, in any other instance, all letters should be lower case. In either case, proof-of-work should be hyphenated between each word.
+
+**Correct usage:**
+
+- Proof-of-work
+- proof-of-work
+
+**Incorrect usage:**
+
+- Proof-of-Work
+- Proof of work
+- proof of work
+
+### Proof-of-stake {#proof-of-stake}
+
+The same rules we apply to proof-of-stake are applicable to proof-of-stake.
+
+**Correct usage:**
+
+- Proof-of-stake
+- proof-of-stake
+
+**Incorrect usage:**
+
+- Proof-of-Stake
+- Proof of stake
+- proof of stake
```

Comment:
Again, just want to double-check this is accurate and proper. @ryancreatescopy You made comments about this in the past, do you recall where you found the proper english rules for these? (also commented about this in #3653)

Reply:
See Ryan's response in discord for context. After his input, I'm fine with keeping this as is if no one opposes it. 

But also happy to change it. My concern is that it becomes wordy and hard to follow.